lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -119,14 +119,16 @@ sub call_rpc {
                     my $tx = $client->tx;
                     $req_storage->{req_url} = $tx->req->url;
                     my $err = $tx->error;
-                    $rpc_failure_cb->(
+                    my $cb_processed_totally;
+                    $cb_processed_totally = $rpc_failure_cb->(
```

Comment:
IIUC we are only using `rpc_failure_cb` in one other call. Why don't we just stop returning anything if a callback is defined? IMO this would make sense for all situations.

Reply:
Yes you are right. Will do it.